### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by OrangeF0x** [shamhi/MemeFiBot#17]
Hello! It's me again. Tell me, is such a number of "zero" clicks the norm?
![image](https://github.com/shamhi/MemeFiBot/assets/166234802/1fa3d2f9-a579-40b6-af9b-5690b7b30c29)

**Comment by shamhi** [comment]
do you update repository on last version?

**Comment by OrangeF0x** [comment]
> Обновляете ли вы репозиторий до последней версии?

Yes, everything was installed literally today, a couple of hours ago

**Comment by MaydayTR38** [comment]
try to delete pycache files

**Comment by OrangeF0x** [comment]
> try to delete pycache files

It doesn't help

**Comment by OrangeF0x** [comment]
**Using a proxy helped me**

**Comment by liluglymane666** [comment]
> **Using a proxy partially helped me. There are more "+" numbers, but there are still a lot of zeros**

have you fixed this?tired of +0

**Comment by OrangeF0x** [comment]
> > **Using a proxy partially helped me. There are more "+" numbers, but there are still a lot of zeros**
> 
> have you fixed this?tired of +0

no, with the proxy there are simply more normal clicks, but +0 is still a lot
